src/lib/holocene/tab/tab.svelte

```diff
@@ -64,13 +64,13 @@
 
 <style lang="postcss">
   .tab {
-    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm hover:border-b-2 hover:border-blue-700 md:text-base;
+    @apply flex cursor-pointer items-center gap-1 whitespace-nowrap border-b-2 text-sm outline-none hover:text-brand focus-visible:rounded-sm focus-visible:shadow-focus focus-visible:shadow-indigo-300 dark:text-secondary hover:dark:text-brand;
```

Comment:
The shadow should be indigo-400 according to the designs. Also I think that because you already have `hover:text-brand` you don't need `hover:dark:text-brand`.